docs/notes.md

```diff
@@ -603,4 +603,254 @@ sends a pair \\((L\_j,R\_j)\\) of points at each step \\(j = k\dots1\\). An
+         &+   {\langle {\mathbf{a}}\_{(j)L}, {\mathbf{a}}\_{(j)R} \circ {\mathbf{y}}^{n}\_{(j)} \rangle} 
+\end{aligned}
+\\]
+
+Combining inner products
+------------------------
+
+We combine the terms in the preceding statement into a single inner product, using the same technique as in the single-value range proof. We will not reproduce the math here since it is the same as in the [combining inner products](index.html#combining-inner-products) step of the single-value proof. Here is the end result:
+
+\\[
+\begin{aligned}
+ \delta_{(j)}(y,z) &= (z - z^{2}) \cdot {\langle {\mathbf{1}}, {\mathbf{y}}^{n}\_{(j)} \rangle} - z^{3} z_{(j)} \cdot {\langle {\mathbf{1}}, {\mathbf{2}}^{n} \rangle}\\\\
+ z^{2}z_{(j)} \cdot v_{(j)} + \delta_{(j)}(y,z) &= {\langle {\mathbf{a}}\_{(j)L} - z {\mathbf{1}}, {\mathbf{y}}^{n}\_{(j)} \circ ({\mathbf{a}}\_{(j)R} + z {\mathbf{1}}) + z^{2} z_{(j)} \cdot {\mathbf{2}}^{n} \rangle}
+\end{aligned} 
+\\]
+
+Blinding the inner product
+--------------------------
+
+The prover chooses vectors of blinding factors \\( \mathbf{s}\_{(j)L}, {\mathbf{s}}\_{(j)R} \\), and uses them to construct the blinded vector polynomials \\(\mathbf{l}\_{(j)}(x), \mathbf{r}\_{(j)}(x)\\). We will not reproduce the steps or the explanation here since it is the same as in the [blinding the inner product](index.html#blinding-the-inner-product) step of the single-value proof. Here are the final equations for the vector polynomials:
+
+\\[
+\begin{aligned}
+  {\mathbf{l}}\_{(j)}(x) &= ({\mathbf{a}}\_{(j)L} + {\mathbf{s}}\_{(j)L} x) - z {\mathbf{1}} & \in {\mathbb Z\_p}[x]^{n}  \\\\
+  {\mathbf{r}}\_{(j)}(x) &= {\mathbf{y}}^{n}\_{(j)} \circ \left( ({\mathbf{a}}\_{(j)R} + {\mathbf{s}}\_{(j)R} x\right)  + z {\mathbf{1}}) + z^{2} z_{(j)} {\mathbf{2}}^{n} &\in {\mathbb Z\_p}[x]^{n} 
+\end{aligned}
+\\]
+
+Proving that \\(t(x)\\) is correct
+----------------------------------
+
+Proving that \\(t\_{(j)}(x)\\) is correct means proving that
+\\({\mathbf{l}}\_{(j)}(x)\\), \\({\mathbf{r}}\_{(j)}(x)\\) are correctly formed, and that
+\\(t\_{(j)}(x) = {\langle {\mathbf{l}}\_{(j)}(x), {\mathbf{r}}\_{(j)}(x) \rangle}\\).
+
+We can combine the statements about \\(t\_{(j)}(x)\\), \\({\mathbf{l}}\_{(j)}(x)\\), and \\({\mathbf{r}}\_{(j)}(x)\\) from all `m` parties in the following manner:
+
+\\[
+\begin{aligned}
+  t(x) &= \sum_{j=0}^{m-1} t\_{(j)}(x)\\\\
+  {\mathbf{l}}(x) &= {\mathbf{l}}\_{(0)}(x) || {\mathbf{l}}\_{(1)}(x) || \dots || {\mathbf{l}}\_{(m-1)}(x) \\\\
+  {\mathbf{r}}(x) &= {\mathbf{r}}\_{(0)}(x) || {\mathbf{r}}\_{(1)}(x) || \dots || {\mathbf{r}}\_{(m-1)}(x) \\\\
+\end{aligned}
+\\]
+
+We can add the \\(t_{(j)}(x)\\) values together to create \\(t(x)\\) instead of taking a linear combination of \\(t_{(j)}(x)\\) values, because each \\(t_{(j)}(x)\\) is calculated with the \\(\mathbf{y}^n\_{(j)}\\) and \\(z_{(j)}\\) challenge variables that are unique to that party`j`, so all of the \\(t_{(j)}(x)\\) values will be offset from one another.
```

Comment:
should be "random linear combination" rather than "linear combination" since adding together is also a linear combination

Reply:
👍